internal/synclet/sidecar/sidecar.go

```diff
@@ -0,0 +1,38 @@
+package sidecar
+
+import (
+	"fmt"
+
+	"k8s.io/api/core/v1"
+)
+
+var syncletPrivileged = true
```

Comment:
We don't seem to set this anywhere yet, and I don't know how we're going to use this, so I don't know what the motivations behind making this var are, but can this be a const or a func param?

Reply:
done!